mono/metadata/marshal-ilgen.c

```diff
@@ -575,7 +576,7 @@ emit_ptr_to_object_conv (MonoMethodBuilder *mb, MonoType *type, MonoMarshalConv
 	}
 }
 
-static gpointer
+static MonoJitICallInfo*
 conv_to_icall (MonoMarshalConv conv, int *ind_store_type)
```

Comment:
These could be enums. i.e. replace `&mono_jit_icall_info.` with `MONO_JIT_ICALL_`.
It doesn't change the overall direction.
It might avoid relocations. On NT/amd64 and NT/arm64 it would not avoid relocations.